src/Platform/index.js

```diff
@@ -41,6 +41,11 @@ export const initPlatform = config => {
 
 const getOrSet = (key, params) => (params ? setInfo(key, params) : getInfo(key))
 
+const adSkipTiersEnum = Object.freeze({
+  noSkipNormalSpeed: 'NOSKIP_NORMAL_SPEED',
+  allowSkipHighSpeed: 'ALLOW_SKIP_HIGH_SPEED'
+})
+
```

Comment:
I'm not sure I see the advantage of this ENUM technique (also the exporting it later on, so a dev can do `if (Platform.Advertising.adSkipTier === Platform.Advertising.AdSkipTiers.noSkipNormalSpeed) {}` versus just having (and documenting) the possible string values

Reply:
Hmmm. There's currently two separate features in the `Advertising` namespace:

- IFA-style anonymized ID for advertising. This allows app developers to integrate with our Ad Framework product.
- Ad Tier related methods, which is more of an entitlement / platform product than a preference.

I don't think they belong in Preferences, but i could be convinced we need a new place...

Reply:
The main value is code completion for developers

Reply:
Does this technique actually have some effect in that regard? If so, I suppose that's a low-value add.

Reply:
if the module exports the enum, then developers don't have to remember the various modes we come up with, and can just type `AdSkipTiers.` and see the options pop up.

Good developer experience is our first goal according to the requirements!